### ISSUE TITLE
MycetOS meeting 26-07-2022

### ISSUE THREAD

**Issue by wwjvdsande** [OpenSourceMycetoma/Series-1-Fenarimols#78]
Chair ErasmusMC
Min UCL

Open Source Mycetoma (MycetOS) Meeting
Meeting Zoom link [here](https://uni-sydney.zoom.us/j/99281549497)

Time: Tue 26th June GMT 11:00 (12:00 PM London, 13:00 Amsterdam, 6:00 New York, 20:00 Sydney) Check your calendar invite

Chair: ErasmusMC

Minute taker: UCL

Recording can be found here: 

Agenda

**Series 1** 
- [ ] [@fantasy121](https://github.com/fantasy121) and [@dmitrij176](https://github.com/dmitrij176) would go through the Epichem library to look up compounds which could fill in the holes of the SAR. The link of the library is found https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/65#issuecomment-955848573. These compounds would then be requested and shipped for in vitro screening.
- [ ] Manuscript: @Wilson-Lm submitted his thesis to the plagiarism check with the current manuscript as a draft in it.  [@fantasy121](https://github.com/fantasy121) and [@dmitrij176](https://github.com/dmitrij176) to update characterization data, for the manuscript and for the in vivo compounds. The main data needs to be polished. 

**Series 2**
- [ ]  [@kym834](https://github.com/kym834) to update on series 2 work with SSP and high school students and double check there is no doubling of compounds synthesis from different participating high schools (Australia vs Seven Oaks for eg).

**Series 5**
- [ ] @bebi78 had sent an e-mail to @bendndi whether any samples of benzoxazoles (structurally related to https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/issues/3) might be available from DNDi's other projects. Compounds arrived and new compounds are coming. Database needs to be updated 

**Updates from each lab**
- [ ] University of Sydney
- [ ] UCL
- [ ] ErasmusMC
- [ ] Updates from other labs

**Any other businesses** 
- [ ]  [UNESCO call](https://www.unesco.org/en/articles/unesco-launches-global-call-best-practices-open-science) was submitted by @mattodd

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib I posted the agenda for todays meeting. If I'm correct ErasmusMC is chair and UCL is taking minutes but I was not completely sure. If it is incorrect please let me know and I change it.

**Comment by mattodd** [comment]
Thanks for doing that @wwjvdsande. I'm at a conference (on antibiotics) right now and thought I could make our meeting today, but it turns out I'm speaking at almost exactly the same time. I will try to make it, but I'm not 100%. Can others make it? e.g. @fantasy121 @OpenSourceMycetoma/corecontrib ?

**Comment by MFernflower** [comment]
Sorry that I was unable to attend the meeting

**Comment by MFernflower** [comment]
@bebi78 MW of perfume compounds
added to master sheet @wwjvdsande

**Comment by MFernflower** [comment]
With regards to the idea of metallic antimicrobial drugs - auranofin has
show decent antibiotic activity @bebi78 I wonder if it was in the mmv
boxes? @wwjvdsande

**Comment by bebi78** [comment]
@MFernflower Yes, auranofin is an interesting compound. A group in Brazil used it as part of the MMV box (see attached file). But I could not find it on the list. 
@wwjvdsande We used auranofin as a reference drug but we have only a few drops of a DMSO stock solution left and it is quite expensive. If auranofin hasn't been tested yet, maybe MMV can provide it. 

[auranofin.antifungal.pdf](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/9234060/auranofin.antifungal.pdf)

**Comment by mattodd** [comment]
Closing. Have mentioned auranofin on the new [Loose Ends](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/wiki/Loose-Ends-for-MycetOS-Series-1) page.
